src/esp/assets/ResourceManager.cpp

```diff
@@ -1580,22 +1581,23 @@ bool ResourceManager::loadRenderAssetGeneral(const AssetInfo& info) {
        scene->meshesMaterialsAsArray()) {
     Cr::Containers::Optional<esp::assets::MeshTransformNode>& node =
         nodes[meshMaterial.first()];
-    if (!node)
+    if (!node) {
       continue;
+    }
 
-    // TODO: either drop MeshTransformNode in favor of SceneData or use
-    // Mn::SceneTools::convertToSingleFunctionObjects() when it's exposed
+    // If meshIDLocal != -1 then we have multiple meshes assigned to the same
+    // MeshTransformNode.  We make subsequent meshes children of the first mesh
+    // we've seen, and give them identity trasnforms.
```

Comment:
I'd keep the TODO for dropping MeshTransformNode here, to make it clear this is just a temporary code to work around its limitations (and not, like, code that patches over magnum's wrongdoings :laughing: )